NAS2D/Renderer/Fade.cpp

```diff
@@ -86,13 +86,13 @@ void Fade::draw(Renderer& renderer) const
 }
 
 
-void Fade::setDuration(unsigned int durationInMilliseconds)
+void Fade::setDuration(const std::chrono::milliseconds& newDuration)
 {
-	if (durationInMilliseconds == 0)
+	if (newDuration.count() <= 0LL)
```

Comment:
You can potentially use [`std::chrono::milliseconds::zero()`](https://en.cppreference.com/w/cpp/chrono/duration/zero) here. That would avoid having to convert to the underlying ticks to do the check.
```cpp
if (newDuration <= std::chrono::milliseconds::zero())
```